datahub-web-react/src/app/entity/shared/tabs/Dataset/Schema/__tests__/filterSchemaRows.test.ts

```diff
@@ -0,0 +1,5 @@
+import { filterSchemaRows } from '../utils/filterSchemaRows';
+
+describe('filterSchemaRows', () => {
+    it('should properly filter schema rows');
```

Comment:
?

Reply:
wow i started this test and completely forgot about it - filling this out rn